common.css

```diff
@@ -0,0 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Audiowide&display=swap");
+
+.nav-logo img {
+  width: 35px;
```

Comment:
Use rem unit when something is greater than 10px as it makes it responsive.

Reply:
I didnt want the image to scale but changed it anyway :)